samples/notebook_instance/README.md

```diff
@@ -0,0 +1,46 @@
+# Notebook Instance Sample
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Update the Notebook Instance Specification
+
+Edit the roleARN value in my-notebook-instance.yaml to include the Sagemaker Execution permissions.
+
+## Using the Notebook Instance operator
+
+### Create a Notebook Instance
+
+This command creates a Sagemaker notebook instance based on the specification provided in my-notebook-instance.yaml.
+The Notebook Instance will start at the Pending state and will transition into InService once ready.
+
+```
+$ kubectl apply -f my-notebook-instance.yaml
+notebookinstance.sagemaker.services.k8s.aws/my-notebook-instance created
+```
+### Update a Notebook Instance
+This commands updates the Notebook Instance with the updated spec provided in my-notebook-instance.yaml. The update command retains the state the controller was previously in. If the update command was called while the controller was in the InService state, it will end up in the InService state after the update. If the update command was called while the controller was in the Stopped/Stopping state, it will end up in the Stopped state after the update.
+Additionally this operator automatically takes care of setting the `Disassociate<item>` fields if the corresponding `<item>` field is empty in the new spec but not empty in the previously applied spec. `<item>` in this case means the following fields: LifecycleConfigName, DefaultCodeRepository, AdditionalCodeRepositories, AcceleratorTypes.
```

Comment:
In the code (PR #82 ) you call this `field` instead of `item`, could you stay consistent here ?

Reply:
done